src/patternfly/components/DataList/data-list.scss

```diff
@@ -280,14 +292,6 @@
     }
   }
 
-  &.pf-m-selected {
-    --pf-c-data-list__item--before--BackgroundColor: var(--pf-c-data-list__item--m-selected--before--BackgroundColor);
-
-    position: relative;
-    z-index: var(--pf-c-data-list__item--m-selected--ZIndex);
-    box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
```

Comment:
We still need to retain the shadow, as that's a pattern we have in other components, too. I think this change might do it?

Add this var

```
--pf-c-data-list__item--item--m-selected--BoxShadow: var(--pf-global--BoxShadow--sm-bottom);
```

And then use it here

```
diff --git a/src/patternfly/components/DataList/data-list.scss b/src/patternfly/components/DataList/data-list.scss
index b531d184..0acec449 100644
--- a/src/patternfly/components/DataList/data-list.scss
+++ b/src/patternfly/components/DataList/data-list.scss
@@ -286,6 +286,10 @@
     position: relative;
     z-index: var(--pf-c-data-list__item--m-selected--ZIndex);
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
+
+    & + & {
+      --pf-c-data-list__item--m-selected--BoxShadow: var(--pf-c-data-list__item--item--m-selected--BoxShadow);
+    }
   }
 
   &.pf-m-ghost-row {
```